README.md

```diff
@@ -64,7 +64,16 @@ These packages will install when you install Terminus.
 
 `*** TBD ***`
 
+## Testing
 
+To run the functional tests:
+
+1. If you need the basics (php, git, direnv, etc): `brew bundle install`
+2. For code coverage: `pecl install pcov`
+3. Prepare a new test site, e.g. make a new multidev in ci-terminus-composer
```

Comment:
The tests currently have the environment hard-coded in most cases, e.g. LocalCommandsTest.php line 64, and line 81 as well. In the tests/Functional directory, when I run ack 'live' | grep 'live' | wc -l gives returns 29, and skimming the results, those matches look like the environment name in every case.

So even if you create a multidev, most of the time it's going to run against the hard-coded environment.

Obviously this needs to be fixed.